packages/client/src/store/modules/users.js

```diff
@@ -79,15 +79,14 @@ export default {
       }
       await dispatch('fetchUsers');
     },
-    async updateEmailSubscriptionPreferences({ dispatch, commit }, { userId, preferences }) {
```

Comment:
All existing uses of this method were being called on the logged in user, plus line 89 below clearly assumes that the method is only called on the logged in user, so I updated the method to only apply to the logged in user.